tests/conftest.py

```diff
@@ -1,56 +1,63 @@
 import pytest
 from pathlib import Path
-import json
 from click.testing import CliRunner
-from genologics_mock.lims import MockLims
-from genologics_mock.entities import (
-    MockArtifact,
-    MockProcess,
-    MockProcessType,
-    MockSample,
-)
-from .helpers import Helpers
+from genologics.lims import Lims
 
+import threading
+import time
 
-@pytest.fixture
-def config():
-    """Get file path to config"""
-
-    return "tests/fixtures/config.yaml"
+from limsmock.server import run_server
 
+PORT = 8000
+HOST = '127.0.0.1'
 
 @pytest.fixture
-def kapa_csv_data():
-    """Get file path to make_kapa_csv_data.json"""
-
-    with open("tests/fixtures/make_kapa_csv_data.json") as file:
-        return json.load(file)
+def server_flat_tests():
+    file_path = f"tests/fixtures/flat_tests"
+    thread = threading.Thread(target=run_server, args=(file_path, HOST, PORT,))
+    thread.daemon = True
+    thread.start()
+    time.sleep(0.1)
 
 
 @pytest.fixture
-def lims():
-    return MockLims()
+def server_test_get_artifacts():
+    file_path = f"tests/fixtures/test_get_artifacts"
+    thread = threading.Thread(target=run_server, args=(file_path, HOST, PORT,))
+    thread.daemon = True
+    thread.start()
+    time.sleep(0.1)
 
 
 @pytest.fixture
-def sample():
-    return MockSample()
+def server_make_kapa_csv():
+    file_path = f"tests/fixtures/test_make_kapa_csv"
+    thread = threading.Thread(target=run_server, args=(file_path, HOST, PORT,))
+    thread.daemon = True
+    thread.start()
+    time.sleep(0.1)
 
 
 @pytest.fixture
-def artifact():
-    return MockArtifact()
-
+def server_make_kapa_csv_missing_udfs():
+    file_path = f"tests/fixtures/make_kapa_csv_missing_udfs"
+    thread = threading.Thread(target=run_server, args=(file_path, HOST, PORT,))
+    thread.daemon = True
+    thread.start()
+    time.sleep(0.1)
 
 @pytest.fixture
-def process():
-    return MockProcess()
+def lims():
+    """Get genologics lims instance"""
+    lims = Lims(f"http://{HOST}:{PORT}", 'dummy', 'dummy')
+    return lims
 
 
-@pytest.fixture(name="helpers")
-def fixture_helpers():
-    """Return a class with small helper functions"""
-    return Helpers()
+@pytest.fixture
+def config():
```

Comment:
Suggest to always use Path objects when working with anything on the file system
```suggestion
def config() -> Path:
```